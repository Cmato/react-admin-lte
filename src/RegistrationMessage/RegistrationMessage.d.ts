import * as React from 'react';

export interface RegistrationMessageProps {
  component?: React.ReactType<RegistrationMessageProps>;
}

declare const RegistrationMessage: React.ComponentType<RegistrationMessageProps>;

export default RegistrationMessage;
