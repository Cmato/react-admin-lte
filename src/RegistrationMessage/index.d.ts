export { default } from './RegistrationMessage';
export * from './RegistrationMessage';
