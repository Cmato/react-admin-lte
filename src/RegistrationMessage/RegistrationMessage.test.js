import React from 'react';
import { shallow } from 'enzyme';
import RegistrationMessage from './RegistrationMessage';

it('renders default', () => {
  expect(shallow(<RegistrationMessage>Sign in</RegistrationMessage>))
    .toMatchSnapshot();
});

it('renders custom component', () => {
  expect(shallow(<RegistrationMessage component="div">Sign in</RegistrationMessage>))
    .toMatchSnapshot();
});
