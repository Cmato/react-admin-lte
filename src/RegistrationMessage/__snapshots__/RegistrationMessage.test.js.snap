// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders custom component 1`] = `
<div
  className="register-box-msg"
>
  Sign in
</div>
`;

exports[`renders default 1`] = `
<p
  className="register-box-msg"
>
  Sign in
</p>
`;
